webpack/envs/clientDevelopmentConfig.js

```diff
@@ -76,29 +69,18 @@ export const clientDevelopmentConfig = {
   },
   name: "force",
   plugins: [
-    new CaseSensitivePathsPlugin(),
```

Comment:
[Supported in 5](https://github.com/Urthen/case-sensitive-paths-webpack-plugin/blob/master/package.json#L42)

Reply:
Not needed with TypeScript

Reply:
There is less reason to bloat our development images and machines with tooling that actively provides little to no value. It also requires monitoring for updates, security risks, etc.

Reply:
This plugin prevents case sensitivity issues between operating systems, guarded at the compiler level. It does not need monitoring for security risks, etc. etc. It has helped us catch issues in the past and you may disagree by creating hypothetical problems out of thin air but that is immaterial to its value. Stop messing with the DX please.

Reply:
Let me double-check that the Webpack warnings created by this plugin have been resolved and I will add it back. It was not compatible with prior versions of Webpack 5.

Reply:
Let me double-check that the Webpack warnings created by this plugin have been resolved and I will add it back. It was not compatible with prior versions of Webpack 5.

Reply:
Working added back! 🎉